### PR TITLE
explanation added to readme for installation with pip

### LINKED ISSUE #34: Installation issues
Hi @CCInc ,

I've tried installing this repo and occured an error while running **./install_openpoints.sh**.

**My versions of software components:**
Using pip in virtual environment.
Python version: Python 3.10.8
pip version: pip 22.3.1
nvcc version: 
Cuda compilation tools, release 11.6, V11.6.124
Build cuda_11.6.r11.6/compiler.31057947_0
gcc version: 7.5.
**OS info:**
linux version: 
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic
kernel version:
5.4.0-125-generic

commands ran:
```bash
#add recursive comment otherwise openpoints folder content is not included (because it's a submodule)
git clone https://github.com/CCInc/3d-ml.git --recursive

#create virtual env with python, go inside it.
cd 3d-ml
python -m virtualenv env_3d
source env_3d/bin/activate

#install pytorch with pip
pip3 install torch torchvision --extra-index-url https://download.pytorch.org/whl/cu116
#install pytorch geo with pip, ${CUDA} = cu116
pip install pyg-lib torch-scatter torch-sparse -f https://data.pyg.org/whl/torch-1.13.0+cu116.html
pip install torch-geometric

#install additional requirements
pip install -r requirements.txt

#install openpoints as root
sudo ./install_openpoints.sh
```

I receive the following warnings
```bash
cuda/emd_kernel.cu(178): error: identifier "CHECK_EQ" is undefined

cuda/emd_kernel.cu(265): error: identifier "CHECK_EQ" is undefined

cuda/emd_kernel.cu(382): error: identifier "CHECK_EQ" is undefined
```
I also receive some warnings:
```bash
.local/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.

.local/lib/python3.10/site-packages/setuptools/command/easy_install.py:160: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
```

What is the recommended way to install this repo if not using conda? 
Also noticed that python <= 3.8 is not supported.

### PR DESCRIPTION
**Pull request by aaronfderybel**
## What does this PR do?

- Fixes #34 
- No breaking changes, only readme has changed.

## Before submitting

- [x] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **test your PR locally** with `pytest` command?
- [ ] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,34 @@ pip install -r requirements.txt
...
+# Below is an example for CUDA 11.6 and torch 1.13
+pip install pyg-lib torch-scatter torch-sparse -f https://data.pyg.org/whl/torch-1.13.0+cu116.html
+pip install torch-geometric
+
+# Install additional requirements
+pip install -r requirements.txt
+
+# If using virtual environments remove the "--user" argument for the last two setup.py commands
+# in the file install_openpoints.sh.
+./install_openpoints.sh
```
Comment (leo-stan) [inline README.md:71]: @CCInc I wonder if we can remove `--user` for conda install as well?

Comment (leo-stan) [reply README.md:71]: We can run with this for now, I will be testing that with the new tests in #41

**Review by leo-stan — APPROVED** [review]
Thanks for the PR!

**Comment by CCInc** [comment]
Thanks for the contribution!

**Pull request merged**